backend/search/utils.py

```diff
@@ -0,0 +1,82 @@
+from opensearchpy.client import OpenSearch
+from django.conf import settings
+import logging
+
+logger = logging.getLogger(__name__)
+
+def update_grant_in_grant_index(grant):
+    client = OpenSearch(
+        hosts = [{'host': settings.OPENSEARCH_URL, 'port': 443}],
+        use_ssl = True,
+        verify_certs = True,
+    )
+    query_body = {
+        "query": {
+            "ids" : {
+                "values" : [12720]
```

Comment:
Looks like a test value is still here... should be something like `grant.id`?

Reply:
Yep! Fixed it. Thanks for catching that.